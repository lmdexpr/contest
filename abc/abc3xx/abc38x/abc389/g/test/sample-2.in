6 924844033
