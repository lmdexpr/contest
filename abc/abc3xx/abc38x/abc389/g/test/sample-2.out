810 2100 3060 3030 2230 1210 450 100 10 0 0
